scripts/bootstrap.sh

```diff
@@ -25,7 +25,7 @@ env >> /tmp/BOOTSTRAP.WHOAMI
 #export PRIVATE_KEY_ARN=${2:-default}
 
 #TODO:
-sudo pip-3.6 --disable-pip-version-check --no-cache-dir install aws-parallelcluster --upgrade
+sudo pip-3.6 --disable-pip-version-check --no-cache-dir install aws-parallelcluster==2.7.0 --upgrade
```

Comment:
Let's make this version a cfn parameter

Reply:
bump to 2.8.0 https://pypi.org/project/aws-parallelcluster/#history

Reply:
Will do on both accounts.